src/feature/control/control.c

```diff
@@ -4470,8 +4470,23 @@ handle_control_hsfetch(control_connection_t *conn, uint32_t len,
     }
   }
 
-  rend_query = rend_data_client_create(hsaddress, desc_id, NULL,
-                                       REND_NO_AUTH);
+  rend_auth_type_t auth_type = REND_NO_AUTH;
+  rend_service_authorization_t *auth = NULL;
+
+  char *cookie = NULL;
```

Comment:
This should be `const`.

Reply:
Fixed in b93f4140f6258d572a9635b06dbfdf7619608b93.